lib/app.coffee

```diff
@@ -74,7 +74,6 @@ resin = require('resin-sdk-preconfigured')
 actions = require('./actions')
 errors = require('./errors')
 events = require('./events')
-plugins = require('./utils/plugins')
```

Comment:
Shouldn't "utils/plugins.coffee" be removed as well?

Reply:
Oooh, well done - that was in the previous PR, but we've since autoconverted it to TS, so when I rebased that change got lost completely. Good spot, fixed :+1: